production/helm/values.yaml

```diff
@@ -8,6 +8,8 @@ serviceAccount:
 
 loki:
   replicas: 1
+  # minReadySeconds: 60
+  # terminationGracePeriodSeconds: 3600
```

Comment:
I think these should be enabled by default and set to the same default that kubernetes has. So `minReadySeconds; 0` and `terminationGracePeriodSeconds: 30`.

Reply:
I used the values from the production example provided by grafana.. https://github.com/grafana/loki/blob/ffe109384ba6124b1dc3011ea859797beb117aa8/production/ksonnet/loki/ingester.libsonnet

I am fine with  setting defaults, i would just like someone from Grafana to verify what the correct defaults are.. @davkal  any recommendations?

Reply:
Set Defaults and removed the if statements.